test/mogrify_test.exs

```diff
@@ -160,6 +160,36 @@ defmodule MogrifyTest do
     File.rm_rf!(path_to_delete)
   end
 
+  @tag :plasma
+  test "plasma success" do
+    path = Path.join(System.tmp_dir(), "1.jpg")
+
+    image =
+      %Image{}
+      |> custom("plasma", "fractal")
+      |> create(path: path)
+
+    assert File.exists?(path) == true
+    assert %Image{path: ^path} = image
+
+    File.rm!(path)
+  end
+
+  @tag :plasma
+  test "plasma binary output buffer matches file output" do
+    image =
+      %Image{}
+      |> custom("seed", 10)
+      |> custom("plasma", "fractal")
+
+    result1 = image |> custom("stdout", "png:-") |> create(buffer: true)
+    result2 = image |> create(path: Path.join(System.tmp_dir(), "1.png"))
```

Comment:
Style readability nit:

```elixir
image_from_stdout = 
  image 
  |> custom("stdout", "png:-") 
  |> create(buffer: true)

image_from_file = 
  image
  |> create(path: Path.join(System.tmp_dir(), "1.png")) 
```

Reply:
Don't think this style change is necessary as it's a copy-paste of the pango test.

Reply:
@talklittle I guess I had too much "style enthusiasm" :slightly_smiling_face: 

Thank you